sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -290,12 +289,8 @@ static gnrc_rpl_parent_t *_gnrc_rpl_find_preferred_parent(gnrc_rpl_dodag_t *doda
             gnrc_rpl_send_DAO(dodag->instance, &old_best->addr, 0);
             gnrc_rpl_delay_dao(dodag);
         }
-        fib_remove_entry(&gnrc_ipv6_fib_table,
-                         (uint8_t *) ipv6_addr_unspecified.u8,
-                         sizeof(ipv6_addr_t));
-        ipv6_addr_t all_RPL_nodes = GNRC_RPL_ALL_NODES_ADDR;
```

Comment:
The `remove` is followed by an  `add` some lines below. I realized that the `add` function does an `update` if the entry exists. So we can save some cycles not removing the entry, because `add` will look for the entry anyways (whether it exists or not)